dev/io.openliberty.java.internal_fat/publish/servers/java17-cdi-server/bootstrap.properties

```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2022 IBM Corporation and others.
+# Copyright (c) 2023 IBM Corporation and others.
```

Comment:
```suggestion
# Copyright (c) 2022,2023 IBM Corporation and others.
```
The original copyright year cannot be replaced unless it was incorrect.

Reply:
Good catch, thanks for pointing that out.  Fixed.